ratelimit/backends/redis.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         self.lua_script: Script = self._redis.register_script(SCRIPT)
 
     async def set_block_time(self, user: str, block_time: int) -> None:
-        await self._redis.set(f"blocking:{user}", True, block_time)
+        await self._redis.set(f"blocking:{user}", 1, block_time)
```

Comment:
the only thing to watch carefully maybe is this, I didnt dig into aioredis very carefully but the encode / decode is performed somewhat differently from other library if I got it correctly reading the arq PR to try to switch from aioredis 1.3.1 to aioredis 2.0

Reply:
hhh, we only need TTL that the key. The value can be anything.